src/theme/components/checkbox.ts

```diff
@@ -0,0 +1,68 @@
+import { checkboxAnatomy } from "@chakra-ui/anatomy";
+import { createMultiStyleConfigHelpers, defineStyle } from "@chakra-ui/react";
+
+const { definePartsStyle, defineMultiStyleConfig } =
+  createMultiStyleConfigHelpers(checkboxAnatomy.keys);
+
+const baseStyleControl = defineStyle(() => {
+  return {
+    borderRadius: "2px", // change the border radius of the control
+    border: "2px solid",
+    borderColor: "gray.700",
+    backgroundColor: "white",
+    color: "white",
+    _checked: {
+      borderColor: "primary.500",
+      backgroundColor: "primary.500",
+    },
+    _hover: {
+      borderColor: "brand.800",
+      backgroundColor: "brand.800",
+      color: "white",
+    },
+    _disabled: {
+      backgroundColor: "gray.300",
+      borderColor: "gray.400",
+      color: "gray.400",
+    },
+  };
+});
+
+const baseStyleContainer = defineStyle({});
+
+const baseStyleLabel = defineStyle({});
+
+const baseStyleIcon = defineStyle({});
+
+// Combine the 4 components of the checkbox
+const baseStyle = definePartsStyle(() => ({
+  icon: baseStyleIcon,
+  container: baseStyleContainer,
+  control: baseStyleControl(),
+  label: baseStyleLabel,
+}));
+
+const sizes = {
+  sm: definePartsStyle({
+    control: { boxSize: "24px" }, // this value includes the border, making it a 20px box with a 2px border
```

Comment:
`height` and `width`, which `boxSize` is a shorthand for, uses the `sizes` tokens (https://chakra-ui.com/docs/styled-system/style-props#layout-width-and-height). The tokens for 24px and 28px (line 51) would be `6` and `7`, respectively. My bad for not clarifying how and where to use these tokens earlier - Figma doesn't show these token values but we want to convert from the pixel or rem values you see in Figma to these tokens.

Reply:
Got it.